release notes/v0.52.0.md

```diff
@@ -0,0 +1,141 @@
+
+Find further details in [#3772](https://github.com/grafana/k6/issues/3772) and [#3773](https://github.com/grafana/k6/issues/3773).
+
+### Panics are no longer being captured [#3777](https://github.com/grafana/k6/pull/3777)
+
+Since this release, Go panics are no longer being captured by k6. This means that if a panic occurs while running a test,
+the k6 process will crash, and the panic stack trace will be printed to the console.
+
+We decided to change this behavior because it's something that was left from the past, as a safeguard, but it's not as
+good as it might seem, as for most cases with multiple goroutines/async it is not enough, and also makes a bunch of potential
+bugs seem like less of an issue.
+
+Thus, this will help us to identify and fix bugs earlier, improve the overall stability of k6, and
+most likely make the experience of developing k6 extensions friendlier.
+
+### `lib.State` no longer has `Group` [#3750](https://github.com/grafana/k6/pull/3750)
+
+As the result of refactoring the implementation of `group` and `check` methods, in order to decouple them, and thus 
+enable other future improvements, the `lib.State` object no longer has a `Group` field.
+
+This change should not affect most users, except for a couple of extensions, for which the use of `Group` was
+already questionable:
+- `xk6-fasthttp`
+- `xk6-g0`
+
+### The `web-dashboard` built-in output becomes `dashboard` [xk6-dashboard#163](https://github.com/grafana/xk6-dashboard/pull/163)
+
+Since `v0.49.0`, k6 includes [xk6-dashboard](https://github.com/grafana/xk6-dashboard) as a built-in output extension.
+The name of the built-in module is the same as was the output name (`web-dashboard`) registered by the extension. 
+Therefore, if you try to use it as an extension (for example, a newer version), you will get the following error:
+
+```sh
+k6 run -o web-dashboard script.js
+
+ERRO[0000] invalid output extension web-dashboard, built-in output with the same type already exists
+```
+
+So, from now on, the registered name of the extension is again the one previously used: `dashboard`. 
+Thus, it is possible to use a version of the [xk6-dashboard](https://github.com/grafana/xk6-dashboard) that is newer than
+the one integrated in k6. In this case, the `dashboard` name must be used as the output name.
+
+### Other breaking changes
+
+- [browser#1318](https://github.com/grafana/xk6-browser/pull/1318) The `Mouse.up` and `Mouse.down` methods no longer take x and y coordinates. Instead, they dispatch events on the current mouse position.
+
+## New features
+
+### Experimental support for TypeScript and ES6+ using esbuild [#3738](https://github.com/grafana/k6/pull/3738)
+
+This release of k6 introduces experimental support for TypeScript and ES6+ using esbuild, thanks to a new
+[compatibility mode](https://grafana.com/docs/k6/latest/using-k6/javascript-compatibility-mode/) named `experimental_enhanced`.
```

Comment:
```suggestion
[compatibility mode](https://grafana.com/docs/k6/latest/using-k6/javascript-typescript-compatibility-mode/) named `experimental_enhanced`.
```

I suggested we change the URL/title to include "TypeScript", but waiting for feedback on the PR for these docs changes if this makes sense or not. If we don't, you can ignore this comment. 🙇